src/constants/MarkdownFiles/posts/2025-08-02-gsoc-25-mostlyk-week09.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "GSoC '25 Week 9 Update by Krish Pandya"
+excerpt: "Hello World, Radio Palettes, and PyPI Release!"
+category: "DEVELOPER NEWS"
+date: "2025-08-03"
+slug: "2025-08-03-gsoc-25-mostlyk-week09"
+author: "@/constants/MarkdownFiles/authors/krish-pandya.md"
+tags: "gsoc25,sugarlabs,week09,mostlyk"
+image: "assets/Images/GSOC.png"
+---
+
+# Week 9: Hello World, Radio Palettes, and PyPI Release!
+
+**Project:** [GTK4 Exploration](https://summerofcode.withgoogle.com/programs/2025/projects/rsHsYZKy)  
+**Mentors:** [Walter Bender](https://github.com/walterbender), [Ibiam Chihurumnaya Ibiam](https://github.com/chimosky), [Juan Pablo Ugarte](https://github.com/xjuan)  
+**Reporting Period:** July 29, 2025 – August 3, 2025
+
+---
+
+## The Week of Debugging, Demos, and Downloads
+
+This week was a classic GSoC rollercoaster: the codebase is now big enough that tiny bugs become huge blockers, and huge features sometimes just work. I spent hours chasing down a path bug that turned out to be a config detail and later on hard coded the paths for a workaround for now , radio tool buttons, palettes, and the first working Hello World activity are all live!
+
+_Hello-World isn't pushed yet, will push it with bundle itself_
+
+
+## Debugging
+
+Early in the project, writing code was fast, just a few graphical interfaces, no integration headaches. Now, every little bug can block the whole flow. This week, I spent hours just figuring out why I couldn't get the art icons working properly, the button signals not ending like I wanted. Turns out, hardcoding the path was the way I have made it work, not elegant, but it works!
+
+A special shoutout to the [Stack Overflow](https://stackoverflow.com/questions/72303475/gtk4-gestureclick-no-released-signal-emitted) and [GNOME Discourse](https://discourse.gnome.org/t/gtk4-need-button-pressed-and-released-signals/8506) threads that helped me fix signaling in Palettes.
+
+
+
+## Radio Tool Buttons & Palettes: The Big Reveal
+
+Let’s start with the fun stuff: radio tool buttons and radio palettes. After a lot of toolkit upgrades and some annoying GTK integration quirks, I finally got radio tool buttons working with their palettes. You can now select between states (eraser, pencil, etc.), and the palette stays in sync with the tool button. 
+
+Also do note: The color scheme is fully customizable via my config, sooo the rainbowish borders you see in the demo is all me, not GTK4 defaults.
+
+Here’s a quick breakdown of what’s working:
+( This is shown in the video which I have been delaying since the past few weeks! )
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/gbaG9CaJJ-U?si=ukHdaz2HTxdHCd9e" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
YouTube videos render more consistently in our style if you use the code specified in https://github.com/sugarlabs/www-v2/blob/main/src/constants/MarkdownFiles/posts/TESTMarkdownFormat.md#youtube-video-embeds